test/message/util-inspect-error-cause.out

```diff
@@ -1,3 +1,28 @@
+Error: Number error cause
+    at *
+    at *
+    at *
+    at *
+    at *
+    at *
+    at * {
+  [cause]: 42
+}
+Error: Object cause
+    at *
+    at *
+    at *
+    at *
+    at *
+    at *
+    at * {
+  [cause]: {
```

Comment:
why brackets around the "cause"? it's not a symbol property

Reply:
It is a non-enumerable properties. Those are always visualized with brackets.

Reply:
ah, thanks, TIL.

So in my example, it'd show up like `{ [cause]: 'b', x: 'y' }`?

Reply:
![image](https://user-images.githubusercontent.com/8822573/146612971-e4754e1a-87a9-42c3-a251-234e49c6b3b1.png)

Reply:
Looks like also, non-enumerables are always displayed at the end, after all the enumerables?

How does that work for Symbol properties, can be both enumerable or non-enumerable, but also would need to be displayed in brackets?